docs/source/05_api_docs/kedro.extras.datasets.rst

```diff
@@ -35,3 +35,4 @@ Data Sets
     kedro.extras.datasets.spark.SparkJDBCDataSet
     kedro.extras.datasets.text.TextDataSet
     kedro.extras.datasets.yaml.YAMLDataSet
+    kedro.extras.datasets.tensorflow.TensorFlowModelDataset
```

Comment:
i believe we order this alphabetically, so can you put it between `spark` and `text`?